apps/src/studio/studio.js

```diff
@@ -1040,6 +1040,10 @@ Studio.onTick = function() {
     Studio.succeededTime = currentTime;
   }
 
+  if (!animationOnlyFrame) {
+    Studio.executeQueue('whenTouchGoal');
```

Comment:
in playlab, `callHandler` will execute the generated code, but all playlab generated code simply adds commands to queues. Those queues need to be executed separately. We typically call `executeQueue` soon after we potentially add something to a queue.

The code you mention will occasionally skip adding commands to the queue (depends on the `allowQueueExtension` param). For simple things like `whenUp`, there really isn't much point in adding multiple events in the queue (this is like spoiling extra repaint messages in the desktop world). But for other cases, (collisions with items/projectiles), we will queue up multiple commands before executing them.